aptos-move/e2e-move-tests/src/tests/code_publishing.data/pack_compile_bytecode/Move.toml

```diff
@@ -0,0 +1,10 @@
+[package]
+name = "A"
+version = "0.0.0"
+
+[addresses]
+A = "0x2"
```

Comment:
How does compilation know where to get the bytecode from?

Reply:
Yes. The current implementation in the move-package system only supports finding bytecode locally. If `.move` files cannot be found, the CLI will try to retrieve `.mv` files inside either the `source` or the `build` folder. To further facilitate this feature, we need to let the`aptos move download` command to download bytecode from chain.

Reply:
Is it possible to just have aptos move compile does this automatically when users depend on bytecode from on chain and it's not available locally? Or at least make sure they get a nice error message if it's not found locally + update docs to make it clear.

In general, I'd suggest thinking more from the perspective of a developer:
1. How would they specify a dependency on bytecode? Is it confusing or difficult to use?
2. How do they use aptos move compile. What errors can they run into?
3. How do they use aptos move deploy. What errors can they run into?
4. etc.

In general let's make sure dev ex is delightful.

Reply:
That makes sense. I will summarize the current status and possible way to tackle the issues listed here so that we can discuss it more in the next Move meeting?

Reply:
How about you just put together a short notion doc on what the dev flow would look like for depending on bytecode? You can then share for faster async feedback.

Reply:
I'm also interested in how this turns out.  Hopefully where you don't have to compile locally

Reply:
So, I looked at this example and tried it myself.

I think we need to continue on the path to get it to downloading bytecode onchain via it's address.  This is a little confusing in general